tests/Unit/Core/Form/IdentifiableObject/DataProvider/ProductFormDataProviderTest.php

```diff
@@ -607,6 +610,62 @@ private function getDatasetsForCategories(): array
         return $datasets;
     }
 
+    /**
+     * @return array
+     */
+    private function getDatasetsForPackedProducts(): array
+    {
+        $datasets = [];
+
+        $expectedOutputData = $this->getDefaultOutputData();
+        $productData = [
+            'packed_products' => [
+                0 => [
+                    'product_id' => 15,
+                    'productName' => 'wicked snowboard',
+                    'quantity' => 3,
+                    'reference' => 'demo_15',
+                    'combination_id' => 0,
+                    'image' => 'http://myshop.com/img/p/no_picture-small_default.jpg',
+                ],
+                1 => [
+                    'product_id' => 42,
+                    'productName' => 'cool glasses',
+                    'quantity' => 5,
+                    'reference' => 'demo_42',
+                    'combination_id' => 0,
+                    'image' => 'http://myshop.com/img/p/no_picture-small_default.jpg',
+                ],
+            ],
+        ];
+
+        $expectedOutputData['description']['packed_products'] = [
+            0 => [
+                'product_id' => 15,
+                'name' => 'wicked snowboard',
+                'combination_id' => 0,
```

Comment:
Can you please add a use case with combination ID different from zero, this will validate the fact that `unique_identifier` is correctly built

Reply:
done